lambda/src/main/scala/migration/Main.scala

```diff
@@ -1,13 +1,39 @@
 package migration
 
 import org.flywaydb.core.Flyway
+import pureconfig._
+import pureconfig.generic.auto._
+import software.amazon.awssdk.auth.credentials.DefaultCredentialsProvider
+import software.amazon.awssdk.regions.Region
+import software.amazon.awssdk.services.rds.RdsUtilities
+import software.amazon.awssdk.services.rds.model.GenerateAuthenticationTokenRequest
+
+object B extends App {
+  new Main().runMigration()
+}
 
 class Main {
 
+  case class MigrationConfig(host: String, port: Int, url: String, username: String)
+
   def runMigration(): Int = {
-    val url = s"${sys.env("DB_URL")}?ssl=true&sslrootcert=${getClass.getResource("/rds-ca-2019-root.pem").getPath}&sslmode=verify-full"
+    val config = ConfigSource.default.load[MigrationConfig] match {
+      case Left(value) => throw new RuntimeException(value.prettyPrint())
```

Comment:
Is the value self-explanatory if you see it in an error message, or is it worth adding a message to the exception like "Failed to load database migration config"?

Reply:
It's usually something like "Cannot resolve value consignment" or something. I'll put the prefix in anyway as it will make it clearer.